src/Beman/InplaceVector/tests/InplaceVector.test.cpp

```diff
@@ -0,0 +1,88 @@
+#include <Beman/InplaceVector/InplaceVector.hpp>
+
+using namespace beman::InplaceVector;
+
```

Comment:
this is a good start, but obviously there's plenty more cases to cover here.  Since there was a lot of controversy about constexpr would certainly be good to have cases here -- and also for the zero capacity case.